modules/creating-cluster-with-aws-kms-key.adoc

```diff
@@ -0,0 +1,211 @@
+// Module included in the following assemblies:
+//
+// * monitoring/enabling-monitoring-for-user-defined-projects.adoc
+
+:_content-type: PROCEDURE
+[id="creating-cluster-with-aws-kms-key"]
+= Creating a ROSA cluster using a custom AWS KMS key
+
+You can create a {product-title} (ROSA) cluster with a customer-provided KMS key that is used to encrypt both node root volumes and persistent volumes for mounted EBS claims.
+
+.Procedure
+
+. Create a custom AWS customer-managed KMS key by running the following command:
++
+[source,terminal]
+----
+$ KMS_ARN=$(aws kms create-key --region $AWS_REGION --description 'Custom ROSA Encryption Key' --tags TagKey=red-hat,TagValue=true --query KeyMetadata.Arn --output text)
+----
++
+This command saves the Amazon Resource Name (ARN) output of this custom key for further steps.
++
+[NOTE]
+====
+Customers provide the `--tags TagKey=red-hat,TagValue=true` that are required for a customer KMS key.
+====
+
+
+. Verify the KMS key has been created by running the following command:
++
+[source,terminal]
+----
+$ echo $KMS_ARN
+----
+
+. Set your AWS account ID to an environment variable.
++
+[source,terminal]
+----
+$ AWS_ACCOUNT_ID=<aws-account>
```

Comment:
It would be better to use an unified variable like <aws_account_id>

Reply:
@yingzhanredhat Given the frequent re-use of the AWS account ID, it made sense to save it to an environment variable.

Reply:
replaced on line 24:
$ AWS_ACCOUNT_ID=<aws-account>
with
$ AWS_ACCOUNT_ID=${AWS_ACCOUNT_ID}